components/carousel/carousel.component.ts

```diff
@@ -353,6 +357,7 @@ export class NzCarouselComponent implements AfterContentInit, AfterViewInit, OnD
           this.pointerDelta = delta;
           this.isDragging = true;
           this.strategy?.dragging(this.pointerDelta);
+          this.cdr.detectChanges();
```

Comment:
`this.cdr.detectChanges();`
should also be updated when the `isDragging` is false